ptx/sec_deriv_prodquot.ptx

```diff
@@ -440,25 +441,16 @@
   <theorem xml:id="thm_deriv_trig">
     <title>Derivatives of Trigonometric Functions</title>
     <statement>
-      <p>
-        <ol cols="2">
-          <li>
-            <p>
-              <m>\displaystyle \lzoo{x}{\sin(x)} = \cos(x)</m>
 
-      <idx><h>derivative</h><h>trigonometric functions</h></idx>
-
-            </p>
-          </li>
+          <idx><h>derivative</h><h>trigonometric functions</h></idx>
 
+      <p>
+        <ol cols="2">
+          <li><m>\displaystyle \lzoo{x}{\sin(x)} = \cos(x)</m></li>
```

Comment:
The `\displaystyle` is now redundant. I must have missed this one.

Reply:
Maybe missed because I asked you to avoid chapters 1--3? I will remove it, and include that in the next force push.